### PR TITLE
* improved server side perfs (memory and CPU)

### PR DESCRIPTION
**Pull request by pascalchevrel**
- add some caching to the view as calculations are still time intensive
- add text blurb about # of occurrences
- Add a column selector at the top to choose locales

### CODE REVIEW

File: web/js/hide_table_rows.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
+    var $chk = $('#grpChkBox input:checkbox');
```
Comment (TheoChevalier) [inline web/js/hide_table_rows.js:2]: Any reason I’m missing to use $ for variable names? If it’s a convention we have, I totally missed it :)

File: web/js/hide_table_rows.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    var $tbl = $('#words');
+    var $tblhead = $('#words th');
+
+    $chk.prop('checked', false);
+
+    $chk.click(function() {
+        var colToHide = $tblhead.filter('.' + $(this).attr('name'));
+        var index = $(colToHide).index();
+        if(colToHide.css('display') === 'none'){
```
Comment (TheoChevalier) [inline web/js/hide_table_rows.js]: missing space before {

File: web/style/transvision.css [hunk 3]
```diff
@@ -1157,3 +1157,23 @@ fieldset {
     left: 60%;
     z-index: 99;
 }
+
+/* Unlocalized words view */
+#unlocalized fieldset#grpChkBox {
+    border: 1px solid #000;
+    background-color: rgba(255,255,255,0.5);
```
Comment (TheoChevalier) [inline web/style/transvision.css]: missing spaces rgba(255, 255, 255, 0.5);

File: web/js/hide_table_rows.js (hunk 1 above)
Comment (pascalchevrel) [inline web/js/hide_table_rows.js:2]: This is a jQuery code convention:
http://stackoverflow.com/a/553734/525582

Comment (TheoChevalier) [inline web/js/hide_table_rows.js:2]: thanks, makes sense

**Pull request merged**